styles.css

```diff
@@ -0,0 +1,52 @@
+* {
+	box-sizing: border-box;
+}
+
+
+
+
+body {
+	margin: 0;
+	font-family: arial, sans-serif;
+	
+}
+
+nav {
+	background-color: #019dcf;
+	color: #ffffff;
+	font-size: 20px;
+	font-weight: 400;
+	letter-spacing: .05em;
+	padding: 40px 60px;
+	
+}
+
+
+@media screen and (max-width: 500px){
```

Comment:
looks good 👍 , but i believe its good practice to place media queries under the rest of the regular css